skops/card/_model_card.py

```diff
@@ -504,9 +505,10 @@ def __init__(
         self._data: dict[str, Section] = {}
         self._metrics: dict[str, str | float | int] = {}
 
-        self._populate_template(model_diagram=model_diagram)
-
-    def _populate_template(self, model_diagram: bool | Literal["auto"] | str):
+    def populate_template(
```

Comment:
users shouldn't have to call this one, do they?

Reply:
If they didn't, then the card would always be empty, would it not?

Reply:
when I create a card based on a template, I expect at least the default sections to be there.

Reply:
Okay, but then it's not different from what we already have. I think this means that we could add the default section names to the `add_` methods but that's it. This PR could be closed.

Reply:
looks like it, and simplify their code.